packages/graphql/src/api-v6/queryASTFactory/ReadOperationFactory.ts

```diff
@@ -171,6 +174,30 @@ export class ReadOperationFactory {
         return [...attributeFields, ...relationshipFields];
     }
 
+    private getSortFields(
+        entity: ConcreteEntity,
+        sortArguments: { edges: { node: Record<string, "ASC" | "DESC"> } }[] | undefined
+    ): { edge: PropertySort[]; node: PropertySort[] }[] {
```

Comment:
I think for these complex typings it is a bit more readable with `Array`

```suggestion
    ): Array<{ edge: PropertySort[]; node: PropertySort[] }> {
```